### ISSUE TITLE
Version number on .xproject file does not match released version 

### ISSUE THREAD

**Issue by xross** [xmos/lib_i2c#78]
The .xproject file states version 5.0.0dc6fb354094

This has not been maintained, this is likely a general issue with lib_repos.

**Comment by lucianomartin** [comment]
In infr_apps there is a script to update the Eclipse files:
https://github.com/xmos/infr_apps/blob/develop/lib/python/prepare_eclipse_files.py

In lib_i2c/lib_i2c there are 4 files that have not been updated recently:
   - .makefile: it is not used as far as I know, it has only some generic information, but no command to compile any code, its content is:

```
all:
    @echo "** Module only - only builds as part of application **"
 
clean:
    @echo "** Module only - only builds as part of application **"
```

  - .cproject: this file is used for Eclipse in XTC and updated by the script prepare_eclipse_files.py

  -. project: this file is used for Eclipse in XTC and updated by the script prepare_eclipse_files.py

  - .xproject: this file contains info such as lib version, I assume it is used only in Eclipse, but I can't prove it. It is updated by the script prepare_eclipse_files.py as well

When I tried to run the script on my MAC laptop I had an error related to the missing Java version 1.6, as xTimeComposer requires it. This version was released in 2006 and on my machine I have a more recent version.

The complication of using such an old version of Java is that we should add the script prepare_eclipse_files.py to the Jenkins tests if we want to maintain those files, but the machines running the Jenkins test have a more recent version of Java.

As it is using such an old version of Java, I am wondering if it would be best to remove those files rather than to update them, but we need to find out who is using them first and check if it is ok to remove them.
